platform/security/idp/security-idp-plugin/security-idp-plugin-nameidpolicy/src/test/groovy/org/codice/ddf/security/idp/plugin/nameidpolicy/NameIdPolicyPresignPluginSpec.groovy

```diff
@@ -0,0 +1,276 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ **/
+package org.codice.ddf.security.idp.plugin.nameidpolicy
+
+import org.apache.cxf.staxutils.StaxUtils
+import org.apache.wss4j.common.saml.OpenSAMLUtil
+import org.opensaml.core.xml.XMLObject
+import org.opensaml.saml.saml2.core.Attribute
+import org.opensaml.saml.saml2.core.AuthnRequest
+import org.opensaml.saml.saml2.core.NameID
+import org.opensaml.saml.saml2.core.NameIDPolicy
+import org.opensaml.saml.saml2.core.Response
+import org.w3c.dom.Document
+import spock.lang.Specification
+import spock.lang.Unroll
+
+class NameIdPolicyPresignPluginSpec extends Specification {
```

Comment:
❓ Should we have test coverage for the currently unsupported formats you added cases for in the implementation above?

Reply:
I originally did, but they seemed unnecessary and I actually took them out during this review process.